app/models/miq_expression/field.rb

```diff
@@ -39,6 +37,14 @@ def target
     end
   end
 
+  private
+
+  delegate :arel_table, :to => :target
+
+  def arel_attribute
```

Comment:
this method means something in active record world.

Not sure if we can call it something different. OR
if we can call it an the model and not implement it here?